Sources/Contentful/ImageOptions.swift

```diff
@@ -347,3 +345,37 @@ private struct ImageParameters {
     static let quality          = "q"
     static let progressiveJPG   = "fl"
 }
+
+
+// Use CGColor instead of UIColor to enable cross-platform compatibility: macOS, iOS, tvOS, watchOS.
+internal extension CGColor {
+
+    // If for some reason the following code fails to create a hex string, the color black will be
+    // returned.
+    internal func hexRepresentation() -> String {
+        let hexForBlack = "000000"
+        guard let colorComponents = components else { return hexForBlack }
+        guard let colorSpace = colorSpace else { return hexForBlack }
+
+        let r, g, b: Float
+
+        switch colorSpace.model {
+        case .monochrome:
```

Comment:
Do you mean `.greayscale` ([`.monochrome` means black/white, and no gray](https://en.wikipedia.org/wiki/List_of_monochrome_and_RGB_palettes#Monochrome_.281-bit.29))